x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -28,6 +28,7 @@ appender.audit_rolling.layout.pattern = {\
                 %varsNotEmpty{, "apikey.name":"%enc{%map{apikey.name}}{JSON}"}\
                 %varsNotEmpty{, "authentication.token.name":"%enc{%map{authentication.token.name}}{JSON}"}\
                 %varsNotEmpty{, "authentication.token.type":"%enc{%map{authentication.token.type}}{JSON}"}\
+                %varsNotEmpty{, "remote_access":%map{remote_access}}\
```

Comment:
I've been thinking about this one too, and kicking the can on a bigger rename refactor that moves away from `remote_access` in other places. Mostly because I still don't have a good alternative. 

I think `remote_cluster_client` is a little too overloaded because of the node role name. One thought here is this is already nested, so if we have

```
"remote_cluster":{"authentication.type":"REALM","user.name":"QkHzNh", ...}
```

It sort of reads as `remote_cluster.authentication.type` and `remote_cluster.user.name` which seems reasonably intuitive. I agree that `remote_cluster` is better than `remote_access` either way.

Reply:
One other naming idea: `cross_cluster_access` -- it gets rid of the ambiguity of `remote` but also captures a little more explicitly that we're dealing with information about access (i.e., authentication, user info etc), as opposed to just `remote_cluster`. WDYT?

Reply:
`cross_cluster_access` sounds good to me. We can go with it for now.